src/views/Project/index.jsx

```diff
@@ -23,6 +23,7 @@ const tagReposMapping = repositories =>
 
 @hot(module)
 @graphql(issuesQuery, {
+  skip: props => !projects[props.match.params.project].repositories,
```

Comment:
This view will only be loaded when it has a project so I don't think this check is necessary.

Reply:
@helfi92 this is to check that the project has github repository or not so if it doesnt have it wont fetch

Reply:
Ah I see what you mean. In that case, are we able to make it more explicit by using the `includes` i.e., `.includes(props.match.params.project)`?

Reply:
isn't `includes` is a method for Array/String?

Reply:
I was referring to something like `projects.includes(props.match.params.project)`.